scripts/apps/web-publisher/services/pub-api-factory.js

```diff
@@ -0,0 +1,120 @@
+/**
+ * @ngdoc service
+ * @module superdesk.apps.web_publisher
+ * @name PubAPIFactory
```

Comment:
I know this may be counter-intuitive, but we need to call this `pubabi` as it is defined in the module. So `@name pubabi`.